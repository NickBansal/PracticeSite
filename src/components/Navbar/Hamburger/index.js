import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
    position: absolute;
    top: 10px;
    right: 16px;
    &:hover {
        cursor: pointer;
    }
`;

const Line = styled.div`
    height: 7px;
    width: 60px;
    margin: 12px;
    background: #0000008a;
    border-radius: 3px;
`;

const Hamburger = () => (
  <Container>
      <Line />
      <Line />
      <Line />
    </Container>
);

export default Hamburger;
