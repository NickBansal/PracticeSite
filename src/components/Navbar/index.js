import React from 'react';
import styled from 'styled-components';
import Hamburger from './Hamburger';

const StyledNavbar = styled.div`
    width: 100%;
    height: 50px;
    text-align: right;
    position: sticky;
    top: 0px;
`;

const Navbar = () => (
    <StyledNavbar>
        <Hamburger />
    </StyledNavbar>
);

export default Navbar;
