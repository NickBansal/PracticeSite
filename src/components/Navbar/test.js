import React from 'react';
import { render } from '@testing-library/react';
import Navbar from '.';

describe('<Navbar />', () => {
  it('should animate with the correct fadein times', () => {
    const container = render(<Navbar />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
