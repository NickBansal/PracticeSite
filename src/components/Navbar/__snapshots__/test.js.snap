// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should animate with the correct fadein times 1`] = `undefined`;
